src/tracks/BBTrack.cpp

```diff
@@ -212,12 +212,12 @@ void BBTCOView::paintEvent( QPaintEvent * )
 	QLinearGradient lingrad( 0, 0, 0, height() );
 	QColor c;
 	bool muted = m_bbTCO->getTrack()->isMuted() || m_bbTCO->isMuted();
-	
+
 	// state: selected, muted, default, user selected
-	c = isSelected() ? selectedColor() : ( muted ? mutedBackgroundColor() 
-		: ( m_bbTCO->m_useStyleColor ? painter.background().color() 
+	c = isSelected() ? selectedColor() : ( muted ? mutedBackgroundColor()
+		: ( m_bbTCO->m_useStyleColor ? painter.background().color()
 		: m_bbTCO->colorObj() ) );
```

Comment:
I think this is a good candidate for style reformat to make it a bit easier to read. Just an example to show it's 3 layers deep, but I'm not opposed to changing it to this. 😄
```suggestion
	c = isSelected()
		? selectedColor()
		:	(muted
			? mutedBackgroundColor()
			:	(m_bbTCO->m_useStyleColor
				? painter.background().color()
				: m_bbTCO->colorObj()
				)
			)
		;
```

Reply:
I know it's not part of the PR changes, but since it popped up I'm commenting on it.

Reply:
I'll leave it alone in order to minimize unrelated changes.